src/extension/background.js

```diff
@@ -308,7 +308,7 @@ async function updateHelpContent() {
     // add listener for clicks on context menu item
     chrome.contextMenus.onClicked.addListener(async ({ menuItemId }, tab) => {
       if (!tab.url) return;
-      contextMenuActions[menuItemId](tab);
+      actions[menuItemId](tab);
```

Comment:
maybe await and reply?

```suggestion
    chrome.contextMenus.onClicked.addListener(async ({ menuItemId }, tab, sendResponse) => {
      ....
      await actions[menuItemId](tab);
      await sendResponse('ok');
```